mne/io/kit/kit.py

```diff
@@ -905,6 +912,7 @@ def read_raw_kit(
     mrk=None,
     elp=None,
     hsp=None,
+    bad_coils=[],
```

Comment:
This one is actually important. If it's not there, it means someone could have passed `mrk, elp, hsp, stim, ...` and now their behavior changes. So we could do this:
```suggestion
    *,
    bad_coils=[],
```
Even safer would be to add this parameter last in the list, after `standardize_names` (with a `*,` preceding it). That's probably better even though this seems like a reasonable place to have the `*,`.